cmd/genconversion/conversion.go

```diff
@@ -71,12 +72,19 @@ func main() {
 	}
 
 	versionPath := path.Join(pkgBase, group, version)
+	if group == "experimental" {
+		//TODO: we should rename the direcotry /expapi to /experimental, so directory path match groupVerson
+		versionPath = path.Join(pkgBase, "expapi", version)
+	}
 	generator := pkg_runtime.NewConversionGenerator(api.Scheme.Raw(), versionPath)
 	apiShort := generator.AddImport(path.Join(pkgBase, "api"))
 	generator.AddImport(path.Join(pkgBase, "api/resource"))
 	// TODO(wojtek-t): Change the overwrites to a flag.
 	generator.OverwritePackage(version, "")
-	for _, knownType := range api.Scheme.KnownTypes(version) {
+	if *groupVersion == "api/v1" {
```

Comment:
Does other clients also generate conversion function? When I wrote this, I was only concerned with how we format the versions in hack/update-generated-conversion.sh. This seems to be the most neat way.